File: PYTHONGUIA/POO/construirdestruir.py
class Vida:
    def __init__(self):                           # Constructor: nace el objeto
        print("Objeto Creado")
    def __del__(self):                            # Destructor: muere el objeto
        print("Objeto Destruido")

v = Vida()                                        # Crea objeto (llama Constructor)
del v                                             # Fuerza destrucción (llama Destructor)

File: C++/POO/construir_destruir.cpp
#include <iostream>
using namespace std;

class Vida {
public:
    Vida() { cout << "Objeto Creado" << endl; }    // Constructor: nace el objeto
    ~Vida() { cout << "Objeto Destruido" << endl; }// Destructor: muere el objeto
};

int main() {
    Vida v;                                       // Crea objeto (llama Constructor)
    return 0;                                     // Fin scope (llama Destructor)
}
